anta/tools/utils.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+"""
+Toolkit for ANTA.
+"""
+from __future__ import annotations
+
+from typing import Any, Dict
```

Comment:
Please always try to use the built-ins dict, list, etc.
https://peps.python.org/pep-0585/